infinispan-remote/persistence/sql-store/src/main/java/org/infinispan/tutorial/simple/remote/persistence/DBCreator.java

```diff
@@ -0,0 +1,146 @@
+package org.infinispan.tutorial.simple.remote.persistence;
+
+import org.apache.commons.io.IOUtils;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DBCreator {
+
+   static final String JDBC_DRIVER = "org.h2.Driver";
+   static final String USER = "infinispan";
+   static final String PASS = "secret";
+   private String DB_URL;
+   private org.h2.tools.Server server;
+
+   public void startDBServer() {
+      startDBServer("9123");
+   }
+
+   public void startDBServer(String port) {
+      DB_URL = String.format("jdbc:h2:tcp://localhost:%s/~/library;", port);
```

Comment:
Will this work on Windows? We may want to mention to get it working this needs to be changed if not.

Reply:
no idea. could someone test it?

Reply:
I will add a comment to specify that